packages/gestalt/src/Button.css

```diff
@@ -8,7 +8,8 @@
 
 .button {
   composes: borderBox from "./Layout.css";
-  composes: rounded from "./Borders.css";
+  border-radius: 24px;
```

Comment:
@jennyscript opted for not yet introducing the new border radius in `"./Borders.css"` since that should be tackled with the `Box` upgrade.